metaschema-cli/src/main/java/gov/nist/secauto/metaschema/cli/commands/ValidateModuleCommand.java

```diff
@@ -90,16 +93,17 @@ protected ISchemaValidationProvider getSchemaValidationProvider(
   }
 
   private static final class ValidationProvider implements ISchemaValidationProvider {
+    @SuppressWarnings("resource")
     @Override
     public XmlSchemaContentValidator getXmlSchemas(
         @NonNull URL targetResource,
         @NonNull IBindingContext bindingContext) throws IOException, SAXException {
       try (InputStream is = this.getClass().getResourceAsStream("/schema/xml/metaschema-model_schema.xsd")) {
-        List<Source> retval = new LinkedList<>();
-        retval.add(new StreamSource(
+        List<Source> sources = new LinkedList<>();
+        sources.add(new StreamSource(
             ObjectUtils.requireNonNull(is,
                 "Unable to load '/schema/xml/metaschema.xsd' on the classpath")));
-        return new XmlSchemaContentValidator(retval);
+        return new XmlSchemaContentValidator(sources);
       }
```

Comment:
Minor, non-blocking: this is good, but in the future can we consider https://github.com/metaschema-framework/metaschema-java/issues/231 for a later enhancement outside the scope of this review?